style.css

```diff
@@ -1,29 +1,31 @@
-.hljs.line-numbers,
-.hljs.selected-lines {
+.hljs.shcb-code-table {
 	display: table;
 	width: 100%;
 }
 
-.hljs.line-numbers .loc,
-.hljs.selected-lines .loc {
+.hljs.shcb-code-table > .shcb-loc {
 	display: table-row;
 	width: 100%;
 }
 
-.hljs.line-numbers {
+.hljs.shcb-line-numbers {
 	border-spacing: 0;
 	counter-reset: line;
 }
 
-.hljs.line-numbers .loc {
+.hljs.shcb-line-numbers > .shcb-loc {
 	counter-increment: line;
 }
 
-.hljs.line-numbers .loc > span {
+.hljs.shcb-code-table > .shcb-loc > span::after {
+	content: "\200B"; /* Hack to force empty spans to render as table cells */
+}
+
+.hljs.shcb-line-numbers .shcb-loc > span {
 	padding-left: 0.75em;
```

Comment:
Add this https://github.com/westonruter/syntax-highlighting-code-block/commit/0de9d1058e901a7a227f42f3d1c3b37f0b9c46b9 and you'll knock out #78 also

Reply:
Humm. I'm not seeing line wrapping at all anymore.

Reply:
I think we may want to address https://github.com/westonruter/syntax-highlighting-code-block/issues/78#issuecomment-617923236 at this point. If we add `white-space: pre-wrap` to each line, then we'll get the wrapping behavior and adding `display: table-cell` to the span will fix the indentation.

```patch
 .hljs.shcb-code-table > .shcb-loc {
 	display: table-row;
+	white-space: pre-wrap;
 	width: 100%;
 }
```

Reply:
At the moment, I'm having a hard time getting the `<code>` to respect the `<pre>`'s padding. This is because when I use `display: table`, it will disregard a parent's padding for `overflow` and only respect the parent's boundary. Resulting in this behavior:

![image](https://user-images.githubusercontent.com/1246453/80070485-df844e80-84f7-11ea-8b3d-a70c01f20866.png)

If you're open to changing our output from `<pre><code>` to `<pre><div><code>`, then that will let us have wrapping and overflow enabled very easily. Thoughts?

Reply:
Take a look at a503c31a79cbd2c8c4c34f47cb2e76c7de370808 and let me know what you think. Improvements welcome!